src/executables/master_heartbeat.c

```diff
@@ -2091,6 +2145,50 @@ hb_add_ping_host (char *host_name)
     {
       strncpy (p->host_name, host_name, sizeof (p->host_name) - 1);
       p->host_name[sizeof (p->host_name) - 1] = '\0';
+      p->port = 0;
```

Comment:
Nit.
```suggestion
      p->port = -1;
```
Because the TCP port ranges from 0 to 65535 even though the 0 is reserved.

Reply:
OK.

As you already know, 0 also checked by if (port < 1 || port > USHRT_MAX) in the hb_add_tcp_ping_host().